frontend/src/components/Charts/DepthChart/index.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useEffect, useState } from "react"
```

Comment:
Is a good practice in React to always create `index.ts|tsx `files, let me know what do you think